src/components/AbstractModal/AbstractModal.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import Modal from 'react-modal';
+
+const AbstractModal = ({ modalId, className, required, modalIsOpen, children }) => {
+    return (
+        <Modal 
```

Comment:
suggestion: add inline stying to the `<Modal>` like this:

`<Modal
style={{
                content: {
                    width: '60%',
                    height: '60%',
                    margin: '0 auto',
                    padding: '5rem',
                },
                overlay: {
                   background: 'lightblue',
                }
            }}`
>
you don't have to use those styles! Make your own, but adjust the size of the modal, give it some padding and position the close button at least.

Reply:
done